x-pack/test/api_integration/apis/management/index_management/indices.js

```diff
@@ -198,7 +198,9 @@ export default function({ getService }) {
           'ilm', // data enricher
           'isRollupIndex', // data enricher
         ];
-        expect(Object.keys(body[0])).to.eql(expectedKeys);
+        const sortedExpectedKeys = expectedKeys.sort();
+        const sortedReceivedKeys = Object.keys(body[0]).sort();
+        expect(sortedReceivedKeys).to.equal(sortedExpectedKeys);
```

Comment:
I think it'd be great to leave a comment similar to the one I suggested, to explain why we're sorting. Someone reading this code might not have visibility into the race condition that causes non-determinism.